synyx/urlaubsverwaltung
# End of sickpay improvements

### Todos

- [x] Clarify end of sick days mail content
- [x] Validation of `maximumSickPayDays` with legal minimum
- [x] `>= 42` durch `> 42` ersetzen
- [x] Nur einmalig rauschicken
- [x] Nicht nur am 7. Tag sondern wenn <=7 Tage

### How to test

* Set `uv.sick-note.end-of-pay-notification.cron=` to `"0 * * * * *"`
* Start with testdata
* Add sick note with more than `42` days with endDate less or equal to `7` days in future
* See Mail in http://localhost:8025/

fixes: #1727
